sequencing-server/src/lib/batchLoaders/simulatedActivityBatchLoader.ts

```diff
@@ -171,6 +175,9 @@ export interface GraphQLSimulatedActivityInstance<
   ActivityArguments extends Record<string, unknown> = Record<string, unknown>,
   ActivityComputedAttributes extends Record<string, unknown> = Record<string, unknown>,
 > {
+  activity_directive: {
```

Comment:
So even tho this is technically correct I wonder if it makes more sense to get the `plan_id` from `simulation_dataset.simulation.plan.id` since ultimately the sequence is generated for a given simulation dataset.

Reply:
Yeah I wasn't sure where the best place to get the `plan_id` was from, but I can update this!